component/background/StorageService.tsx

```diff
@@ -0,0 +1,92 @@
+import {STORAGE_KEY_ID} from "../client/Constants";
+
+class StorageService {
+  private storage: chrome.storage.StorageArea;
+
+  constructor() {
+    this.storage = chrome.storage.sync;
+    this.storage.get(STORAGE_KEY_ID, (id) => {
+      // If user id is not locally stored, generate a new one.
+      if (!id) this.storage.set({STORAGE_KEY_ID: this.generateUUID()});
+    });
+  }
+
+  /**
+   * Clear the storage space and load with default settings
+   * @return {Promise<{}>} A promise whose resolve takes no argument, which runs after defaults are set
+   */
+  public reset(): Promise<{}> {
+    return new Promise((resolve) => {
```

Comment:
If this is dead code, I would recommend remove it. @zhoutwo.
RIght now, there is only one thing in setting. Reset makes no sense.